src/ipfsd-daemon.js

```diff
@@ -263,7 +275,15 @@ class Daemon {
         }
       },
       data: (data) => {
-        output += String(data)
+        let line = String(data)
```

Comment:
The name `line` seems kind of unrelated. What do you think of `dataString`, or something more related to the string content?

Reply:
See previous comment.